src/plugins/opensearch_dashboards_react/public/exit_full_screen_button/exit_full_screen_button.test.tsx

```diff
@@ -29,36 +29,79 @@
  */
 
 import React from 'react';
-import sinon from 'sinon';
 import { ExitFullScreenButton } from './exit_full_screen_button';
 import { keys } from '@elastic/eui';
-import { mount } from 'enzyme';
+import { shallow } from 'enzyme';
+import { getLogosMock } from '../../../../core/common/mocks';
 
-test('is rendered', () => {
-  const component = mount(<ExitFullScreenButton onExitFullScreenMode={() => {}} />);
-
-  expect(component).toMatchSnapshot();
+const mockProps = () => ({
+  onExitFullScreenMode: jest.fn(),
+  logos: getLogosMock.default,
 });
 
-describe('onExitFullScreenMode', () => {
-  test('is called when the button is pressed', () => {
-    const onExitHandler = sinon.stub();
+describe('ExitFullScreenButton', () => {
+  it("is rendered using the dark theme's mark by default", () => {
+    const props = {
+      ...mockProps(),
+    };
```

Comment:
Unnecessary memory io?

```suggestion
    const props = mockProps();
```